include/boost/geometry/algorithms/detail/buffer/buffer_inserter.hpp

```diff
@@ -202,28 +206,28 @@ struct buffer_range
         typename Collection,
         typename Iterator,
         typename DistanceStrategy,
-        typename SideStrategy,
+        typename SegmentStrategy,
```

Comment:
This is slightly related (because I had to pass it too), and fix the issue that we had two side-strategies (one real, and one to generate the sides along buffer segments). The last one is now renamed to `SegmentStrategy` in this code.